pyteal/ast/router.py

```diff
@@ -288,6 +287,9 @@ class CondNode:
     condition: Expr
     branch: Expr
 
+    def reframe_asts(self, stack_frames: NatalStackFrame) -> None:
```

Comment:
maybe a line about what a reframe is

Reply:
f94fa7a8f5ee7f5356eafb9f462c192c3a605605